knowledge-base/issues-on-connecting-telerik-reporting-standalone-report-designer-to-telerik-report-server.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Issues on connecting Telerik Reporting Standalone Report Designer to Telerik Report Server
+description: Issues on connecting Telerik Reporting Standalone Report Designer to Telerik Report Server.
+type: how-to
+page_title: Issues on connecting Telerik Reporting Standalone Report Designer to Telerik Report Server
+slug: issues-on-connecting-telerik-reporting-standalone-report-designer-to-telerik-report-server
+res_type: kb
+---
+
+## Description
+  
+- Warning message on connecting **Telerik Reporting Standalone Report Designer** to **Telerik Report Server**:  
```

Comment:
Listing several error messages and warning in a single KB is not a good idea as it is hard for google to index everything. In other words, when customers are searching for a single, concrete error, this KB might not come in the search results as other keywords are indexed. The recommendation in such cases is to split the KB in several articles (KBs) - one for each error and warning. This will allow capturing the correct customer context in each of them and applying the required SEO.

I would split this KB in 5 similar troubleshooting KBs, having the same solution, but different titles, descriptions based on the exact error.

Also, when describing the issue, provide more details on where and when the error appears. In what scenarios and use cases?